data/human/deep missions.txt

```diff
@@ -2431,3 +2431,330 @@ event "battle in arneb end"
+			`	Thorleif shakes his head. "Lady, no. We're not.. anyone's employees, nor do we need work here. I am looking for a village, hamlet really, called Grimssalr. It's where my folk come from."`
+			`	The woman frowns for a second then seems to break from a trance as real warmth washes over her face. "I am sorry, it's been so long since I've dealt with people who weren't just looking for work. Alfheim, well, it is what it used to be, but only just more so now." She shrugs, "I know of Grimssalr though. I'm actually from Lindsvoll which is in the same valley."`
+			`	"So can you tell us where it is?" Thorleif asks earnestly. The woman smiles shyly at Thorleif. "Tell you? How about I take you there?"`
+			choice
+				`	(She can come.)`
+					goto accompany
+				`	(We don't need her.)`
+			`	You start to say that it isn't necessary for her to accompany you, when Thorleif cuts you off. "That sounds great!"`
+			`	She smiles broadly at Thorleif and nods.`
+			choice
+				`	(You sigh and shake your head.)`
+			label accompany
+			`	She immediately abandons her kiosk and shrugs. "I didn't care much for that job anyways."`
+			`	You and Thorleif follow her to a local transit hovercraft, where she and Thorleif quickly sit together. You find yourself sitting alone.`
+			`	For the next three hours, it sounds as if they are in deep conversation, while you drift in and out of sleep watching as oil fields give way to rainforests and then rainforests shrink before great and arid mountains.`
+			`	At some point you feel Thorleif shaking your shoulder. "We're here."`
+			choice
+				`	(Yawning, you rise from your warm seat.)`
+			`	The town of Grimssalr is nestled along the foothills of the towering Ettin mountains in western Alfheim. Town is perhaps too generous a word; it is instead more like a couple dozen houses mostly filled with old people long abandoned by their children seeking wealth and adventure elsewhere.`
+			`	The woman, who you've learnt is named Ragnhild, introduces Thorleif to some of the folks here, and before long they have found ways to connect their ancient genealogies together and everyone is suddenly treated as family. You eat their meager fare, swap stories, and listen to legends about the old times.`
+			`	As the sun begins to set, Thorleif approaches you, "It is time," he says. "The journey to the cairn is only a short hike from here. I would that you accompany me. I think Skaldgar would have wanted that."`	
+			choice
+				`	(Leave Grimssalr with Thorleif.)`
+					goto followcairn
+				`	"It is best you do this alone."`
+					goto stayingrim
+			label stayingrim
+			`	Thorleif frowns. "I cannot. It does not feel right. It is only because of you, that I am here at all. I need you by my side to finish this. For me, and for him.`
```

Comment:
```suggestion
			`	Thorleif frowns. "I cannot. It does not feel right. It is only because of you, that I am here at all. I need you by my side to finish this. For me, and for him."`
```